addons/smtp-mail.sh

```diff
@@ -189,7 +189,7 @@ tls_trust_file  /etc/ssl/certs/ca-certificates.crt
 account         $MAIL_SERVER
 host            $MAIL_SERVER
 port            $SMTP_PORT
-from            $MAIL_USERNAME@$MAIL_SERVER
+from            $MAIL_USERNAME
```

Comment:
`$MAIL_USERNAME@$MAIL_SERVER` won't work in all situations.

Reply:
I've tested with LKDCNoini09.com (example) and it works, it just says .invalid after.

Reply:
what I mean is, that I have a mailaddress:
`test@mail.de` and the mailserver address is `smtp.mail.de`
so `$MAIL_USERNAME@$MAIL_SERVER` would produce `test@smtp.mail.de` which is wrong.

Reply:
we need to separate them